scripts/analyze.sh

```diff
@@ -13,7 +13,7 @@ function run_analyze() {
     exit 1
   fi
 
-  result=$(flutter pub run dart_code_metrics:metrics .)
+  result=$(flutter pub run dart_code_metrics:metrics lib test)
```

Comment:
Does it work recursively inside the directories specified then? 🤔
If so I guess we could just give it the absolute path from `pwd` or such.

Reply:
it does, I tested that.
pwd is an interesting idea, not sure if that would work, let me give it a try